lib/utils/report.mjs

```diff
@@ -175,14 +175,13 @@ function isFixApplied(problem) {
 	const range = hasRange ? { start, end } : node?.rangeBy({ index, endIndex });
 
 	if (typeof fix !== 'function') {
-		if (hasRangeData) addFixData({ fixersData, ruleName, range, fixed: false });
```

Comment:
[ask] Again, I wonder if `FixerData` is necessary when `fixed` is `false`. Then, `FixerData.range` should be undefined, right?

```js
{
  'function-name-case': [
    {fixed: false, range: undefined}
  ]
}
```

Can you explain how such data would be used?

Reply:
It will be clearer with the tests.
i.e. `if (hasRangeData)` will be removed